src/lib/viewers/controls/media/MediaSettings/__tests__/MediaSettingsAutoplay-test.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { mount, ReactWrapper } from 'enzyme';
+import MediaSettings, { Context, Menu } from '..';
+import MediaSettingsAutoplay from '../MediaSettingsAutoplay';
+
+describe('MediaSettingsAutoplay', () => {
+    const getContext = (): Partial<Context> => ({ setActiveMenu: jest.fn() });
+    const getWrapper = (props = {}, context = getContext()): ReactWrapper =>
+        mount(<MediaSettingsAutoplay autoplay onAutoplayChange={jest.fn()} {...props} />, {
+            wrappingComponent: MediaSettings.Context.Provider,
+            wrappingComponentProps: { value: context },
+        });
+
+    describe('event handlers', () => {
+        test('should surface the selected item on change', () => {
```

Comment:
Should this test the `false` option as well?

Reply:
I'm not sure it's necessary, since the underlying logic is the same. Thoughts?

Reply:
I guess in terms of testing event handlers, that true. Just thought a test to assert the expected options in the menu might be good